Lib/fontTools/varLib/__init__.py

```diff
@@ -763,19 +824,22 @@ def load_designspace(designspace):
 	normalized_master_locs = [models.normalizeLocation(m, internal_axis_supports) for m in internal_master_locs]
 	log.info("Normalized master locations:\n%s", pformat(normalized_master_locs))
 
-	# Find base master
-	base_idx = None
+	# Find default master after normalization.
+	# TODO: Can the base master ever be anything other than what ds.findDefault() returns?
+	default_master_index = None
 	for i,m in enumerate(normalized_master_locs):
 		if all(v == 0 for v in m.values()):
-			assert base_idx is None
-			base_idx = i
-	assert base_idx is not None, "Base master not found; no master at default location?"
-	log.info("Index of base master: %s", base_idx)
+			if default_master_index is not None:
+				raise VarLibValidationError(f"Designspace {ds_path}: More than one default master found.")
+			default_master_index = i
+	if default_master_index is None:
+		raise VarLibValidationError(f"Designspace {ds_path}: Default master not found after normalizing the locations; no master at default location? Check the mappings.")
```

Comment:
"There should be a master/source at the default location, meaning that the location of that master along each axis is equal to the default value for that axis" or something like that